src/App.tsx

```diff
@@ -1,204 +1,62 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+/* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
try to rewrite without disabling the linter

Reply:
i tried to add USER_ID in dependancy array

  useEffect(() => {
    getTodos(USER_ID)
      .then(setTodos)
      .catch(() => setError(Error.LoadTodos));
  }, [USER_ID]);
  
  but it didn't help, i don't know how to fix it

Reply:
because it should be state or prop in array of deps